src/components/Person/Person.jsx

```diff
@@ -1 +1,37 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    sex,
+    age,
+    partnerName,
+    isMarried,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age !== undefined && (
```

Comment:
you can just write simplier like ```{age && (```
because this value is automatically converted to boolean and if it is true than the next condition will work